library/analytics/src/main/java/org/cru/godtools/analytics/AnalyticsService.java

```diff
@@ -55,6 +61,11 @@
     private static final String ADOBE_NOT_LOGGED_IN = "not logged in";
     private static final String ADOBE_GODTOOLS = "GodTools";
 
+    /* SnowPlow value constants */
+    private static final String SNOWPLOW_APP_ID = "GodTools";
+    private static final String SNOWPLOW_NAMESPACE = "GodToolsSnowPlowAndroidTracker";
+    private static final String SNOWPLOW_ENDPOINT = "snowplow.cru.org";
```

Comment:
is there any desire to keep development tracking separate from release tracking?

Reply:
There will be a stage environment, so we may want to do that for development. Currently there is one environment.

Reply:
alright, let's put these as build config in the build script. let me find you a reference :)

Reply:
well, at least the endpoint parameter :)

Reply:
https://github.com/CruGlobal/godtools-android/blob/develop/library/analytics/build.gradle#L9
https://github.com/CruGlobal/godtools-android/blob/develop/library/analytics/src/main/java/org/cru/godtools/analytics/AnalyticsService.java#L76